### PR TITLE
OHRI-543: Remove 'Selected Items' label from multi-select input field

### PR DESCRIPTION
**Pull request by ODORA0**

### CODE REVIEW

File: src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx [hunk 1]
```diff
@@ -104,8 +104,7 @@ export const OHRIMultiSelect: React.FC<OHRIFormFieldProps> = ({ question, onChan
             invalidText={errors[0]?.errMessage}
           />
         </div>
-        <div className={styles.formField}>
-          <OHRILabel value={'Selected Items'} />
+        <div className={styles.formField} style={{ marginTop: '.1%' }}>
```
Comment (ebambo) [inline src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx:107]: @pirupius am not an expert on this but I think this `.1%` is not a good practice, right? Instead here, I think @ODORA0 should use `xx rem` and follow the designs. Please keep an eye on this.

**Review by ebambo — CHANGES_REQUESTED** [review]

File: src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx (hunk 1 above)
Comment (pirupius) [reply src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx:107]: @ebambo i had a similar concern so am working with him to resolve this

**Review by pirupius — APPROVED** [review]

File: src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx (hunk 1 above)
Comment (ODORA0) [reply src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx]: @eudson @pirupius Changed to a more responsive rem

Comment (ebambo) [reply src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx]: Thank you guys

**Pull request merged**